po/common/fr.po

```diff
@@ -4380,15 +4380,15 @@ msgstr "Texte ou phrases indiquant l'origine du produit et / ou de ses ingrédie
 
 msgctxt "nutriscore_grade_producer_note"
 msgid "Nutri-Score grade from A to E displayed on the product label"
-msgstr "Note Nutri-Score de A à E affichée sur l'étiquette du produit"
+msgstr "Note Nutri-Score de A à E affichée sur l'étiquette produit"
 
 msgctxt "nutriscore_grade_producer_import_note"
 msgid "Open Food Facts computes the Nutri-Score grade based on the information provided (nutrition facts and category). If the grade we compute is different from the grade you provide, you will get a private notification on the producers platform so that the difference can be resolved."
 msgstr "Open Food Facts calcule le Nutri-Score selon les informations fournies (catégorie et informations nutritionnelles). Si la note que nous calculons est différent de la note que vous fournissez, vous recevrez une notification privée sur votre plateforme producteur afin que nous puissions résoudre cette disparité."
 
 msgctxt "nutriscore_score_producer_note"
 msgid "Nutri-Score score (numeric value from which the A to E grade is derived)"
-msgstr "Score Nutri-Score (valeur numérique à partir de laquelle la note de A à E est attribuée)"
+msgstr "Note Nutri-Score (valeur numérique de laquelle est dérivée le rang A à E)"
```

Comment:
I can't find those strings in crowdin, I'd like to change some of the French translations...

https://translate.openfoodfacts.org/translate/openfoodfacts/440/en-fr?filter=basic&value=0#q=nutriscore_score_producer_note

Reply:
"X-Crowdin-File-ID: 1378\n"
https://translate.openfoodfacts.org/translate/openfoodfacts/1378/en-fr?filter=basic&value=0#q=nutriscore_score_producer_note